packages/charts-react-axis/src/LeftAxis.js

```diff
@@ -1,53 +1,70 @@
 import React, { PropTypes } from 'react';
 import YTick from './YTick';
 
-const pathStyle = {
-  fill: 'none',
-  strokeWidth: 1,
-  shapeRendering: 'crispEdges',
+/* eslint-disable react/display-name */
+const calcLabels = (max, min, tickCount) => {
+  let labels = [];
+  for (let i = tickCount; i > -1; i--) {
+    labels.push(i * (max - min) / tickCount + min);
+  }
+  return labels;
 };
 
-/* eslint-disable react/display-name */
-const renderTickWith = (interval) => (_, i) => {
-  const offset = interval * (i + 1);
+const renderTickWith = (interval, labels, width) => (_, i) => {
+  const offset = interval * i;
 
   return (
     <YTick
       key={i}
-      label="Label"
+      label={labels[i].toString()}
       offset={offset}
+      width={width}
     />
   );
 };
 /* eslint-enable react/display-name */
 
 const LeftAxis = (props, context) => {
```

Comment:
Looks like `BottomAxis` and `LeftAxis` classes have a lot of common functions and properties. Does it make sense to have one class that accepts Axis type (`left` or `bottom`) or maybe one generic class on top?